docs/index.asciidoc

```diff
@@ -17,7 +17,7 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 === Logstash Integration Plugin
 
-// include::{include_path}/plugin_header.asciidoc[]
```

Comment:
@karenzone I have enabled back since we excluded the plugin from docs generation.
If you will have a time, I would appreciate your quick eye on docs wordings as well.
Thank you!